linux_os/guide/system/network/network-nftables/set_nftables_table/rule.yml

```diff
@@ -0,0 +1,32 @@
+documentation_complete: true
+
+prodtype: sle15
+
+title: 'Ensure a table exists for nftables'
+
+description: |-
+   Tables in nftables hold chains. Each table only has one address family and only applies 
+   to packets of this family. Tables can have one of six families.
+
+rationale: |-
+    Nftables doesn't have any default tables. Without a table being build, nftables will not filter
+    network traffic.
+    <b>Impact</b>Adding rules to a running nftables can cause loss of connectivity to the system
+
```

Comment:
Please add warning like [this](https://github.com/ComplianceAsCode/content/blob/1b538dfc48ab5bd43f8d9d9a1b6823707dcdbf71/linux_os/guide/services/ssh/ssh_server/sshd_use_strong_rng/rule.yml#L55) about risk of losing connectivity to the system.